tests/visible.lua

```diff
@@ -0,0 +1,165 @@
+local lu = require("luaunit")
+local vec = require("vector")
+local vis = require("visible")
+
+
+function testOrientation()
+  lu.assertTrue(vec.alignment(vec(0, 0), vec(1, 0), vec(0, -1)) > 0)
+  lu.assertTrue(vec.alignment(vec(0, 0), vec(1, 0), vec(0, 1)) < 0)
+  lu.assertTrue(vec.alignment(vec(0, 0), vec(1, 0), vec(2, 0)) == 0)
+  lu.assertTrue(vec.alignment(vec(0, 0), vec(0, -1), vec(0, 1)) == 0)
+  lu.assertTrue(vec.alignment(vec(0, 1), vec(1, 1), vec(2, 1)) == 0)
+end
+
+function testIntersection()
```

Comment:
This seems like the kind of function where you will want to test edge cases. does it work when they exactly intersect at a point? What does it return when they do not intersect at all?